ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -809,7 +809,10 @@ import '../nuxeo-button-styles.js';
       tableSettings.columns = {};
       if (this.columns) {
         this.columns.forEach((column, idx) => {
-          tableSettings.columns[column.field ? column.field : `col-${idx}`] = { hidden: column.hidden };
+          tableSettings.columns[column.field ? column.field : `col-${idx}`] = {
+            hidden: column.hidden,
+            alwaysVisible: column.alwaysVisible,
```

Comment:
I was not expecting us to need this, since we cannot change `alwaysVisible` from the table settings. What is the scenario where we need this?

Reply:
@Gabez0r I thought we needed this for the data-table-settings component - will double-check.